### PR TITLE
Update secure-by-default.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about tenant override taking effect versus Secure by default if MX is not pointing to EOP.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6427

File: microsoft-365/security/office-365-security/secure-by-default.md [hunk 1]
```diff
@@ -51,6 +51,9 @@ Because Microsoft wants to keep our customers secure by default, some tenants ov
 - IP Allow List (connection filtering)
 - Exchange mail flow rules (also known as transport rules)
 
+> [!NOTE]
+> In case, a third party spam filter exists or MX record is not point to EOP, tenant overrides are applied.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/secure-by-default.md]:
```suggestion
> If a third-party spam filter exists or the MX record does not point to EOP, tenant overrides are applied.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@MSFTTracyP Gentle reminder.

**Comment by chrisda** [comment]
Closing, because the addition is confusing and not correct. There is absolutely no way to allow malware (always) or high-confidence phishing messages (recently) into the service, even if you have an override configured. This includes customers with a service or device sitting in front of Microsoft 365.

**Pull request closed without merging**